search/search-domain/src/main/java/io/camunda/search/filter/TenantFilter.java

```diff
@@ -21,6 +22,7 @@ public static final class Builder implements ObjectBuilder<TenantFilter> {
     private Long key;
     private String tenantId;
     private String name;
+    private Long memberKey;
```

Comment:
```suggestion
    private Set<Long> memberKeys;
```
to search by userKey and user's groups keys